_sass/_components/lists.scss

```diff
@@ -35,6 +35,28 @@
 
 // Project page lists
 
-.partner-info h5 {
-    @include u-padding-y(2);
+.partner-info {
+  a {
+    color: color('accent-cool');
+    text-decoration: underline;
+  }
+
+  .usa-sidenav__sublist a {
+    padding-left: 0;
+    text-decoration: underline;
+
+    &:hover {
+      background-color: unset;
+      color: color('primary-dark');
+    }
+  }
+
+  h5 {
+    @include u-padding-top(1);
+    @include u-padding-bottom(2);
```

Comment:
Could we switch these two, so it's `u-padding-top(2)` and `u-padding-bottom(1)`

Reply:
👍